app/controllers/registration_controller.rb

```diff
@@ -12,17 +12,20 @@ def tracksheet
     render :plain => pdf.render, :content_type => :pdf
   end
 
-  def workflow_list
-    docs = Dor::SearchService.query(%(id:"#{params[:apo_id]}"))['response']['docs']
+  def workflows_for_apo(apo_id)
+    docs = Dor::SearchService.query(%(id:"#{apo_id}"))['response']['docs']
```

Comment:
I'm not sure I understand why we do this with weird solr indirection, especially when we load the object below anyway?